### PR TITLE
Add a Birthday Challenge Ranking page

### PR DESCRIPTION
**Pull request by luap42**
Shhhhhh.

### CODE REVIEW

File: app/controllers/birthday_controller.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class BirthdayController < ApplicationController
+  def index
+    render layout: 'without_sidebar'
+  end
+
+  def ranking
+    @ranking = Rails.cache.fetch('birthday-ranking', expires_in: 1.hour) do
+      User.unscoped.joins(:posts).where('posts.score > 0.6') \
```
Comment (ArtOfCode-) [inline app/controllers/birthday_controller.rb:8]: `>` or `>=`?

File: public/assets/undraw_happy_birthday_s72n.svg:Zone.Identifier [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
+ZoneId=3
+HostUrl=about:internet
```
Comment (ArtOfCode-) [inline public/assets/undraw_happy_birthday_s72n.svg:Zone.Identifier:3]: Um?

File: app/controllers/birthday_controller.rb (hunk 1 above)
Comment (luap42) [reply app/controllers/birthday_controller.rb:8]: > is correct per an internal discussion. (we don't want exactly 1 upvote to suffice)

File: public/assets/undraw_happy_birthday_s72n.svg:Zone.Identifier (hunk 2 above)
Comment (luap42) [reply public/assets/undraw_happy_birthday_s72n.svg:Zone.Identifier:3]: good question...

**Pull request merged**